docker/api/container.py

```diff
@@ -392,7 +393,8 @@ def create_container(self, image, command=None, hostname=None, user=None,
                 version 1.10. Use ``host_config`` instead.
             dns_opt (:py:class:`list`): Additional options to be added to the
                 container's ``resolv.conf`` file
-            volumes (str or list):
+            volumes (str or list): List of paths inside the container to use
```

Comment:
ah, nevermind, we just add it to a one-element list if it's a string. Fine by me.